Exercício 1 versão 2

```diff
@@ -0,0 +1,111 @@
+Exercício 1 versão 2
+
+Início
+levantar (sofá)
+andar (banheiro)
+se 'porta do banheiro' = fechada 
```

Comment:
se isso for verdadeiro , seu algoritmo manda abrir a porta
e aí vai pra linha 11
logo, vc abriu a porta mas não entrou no banheiro
corrigir essa lógica

Reply:
corrigido